pkg/cmd/target/unset.go

```diff
@@ -62,7 +64,12 @@ func runCmdUnset(f util.Factory, o *UnsetOptions) error {
 	case TargetKindShoot:
 		targetName, err = manager.UnsetTargetShoot()
 	case TargetKindControlPlane:
-		targetName, err = manager.UnsetTargetControlPlane()
+		err = manager.UnsetTargetControlPlane()
+		if err == nil {
+			var currentTarget target.Target
+			currentTarget, err = manager.CurrentTarget()
+			targetName = currentTarget.ShootName()
+		}
```

Comment:
```suggestion
		var currentTarget target.Target
		currentTarget, err = manager.CurrentTarget()
		targetName = currentTarget.ShootName()
			
		err = manager.UnsetTargetControlPlane()
```
WDYT?

Reply:
In both cases the err of `manager.CurrentTarget()` is not handled. 
```suggestion
		currentTarget, err := manager.CurrentTarget(); 
		if err != nil {
		  targetName = "nothing"
		   break
		}
		targetName = currentTarget.ShootName()
		err = manager.UnsetTargetControlPlane()
```

Reply:
I don't understand ... the error is returned after the switch

Reply:
```
currentTarget, err = manager.CurrentTarget()
targetName = currentTarget.ShootName()
```
I assume that currentTarget is `nil` if the `manager.CurrentTarget()` return an error. You cannot call `ShootName()` on a nil pointer.

Reply:
In golang all errors have to handled immediately. We should also never override previous errors and handle only the last one even if all the calls in between would work. If we could ignore an error we should write:
```go
currentTarget, _ = manager.CurrentTarget()
```

Reply:
You are right, of course. I changed it to return the error directly. We could also just not fetch the shoot name in this case as it is only used for the output anyway. But I think if an error occurs something is wrong (file read failed) and the user should get notified.